### PR TITLE
Add support for column comments

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: src/databricks/sqlalchemy/test_local/e2e/test_basic.py [hunk 1]
```diff
@@ -188,6 +188,26 @@ def test_create_table_not_null(db_engine, metadata_obj: MetaData):
     metadata_obj.drop_all(db_engine)
 
 
+def test_column_comment(db_engine, metadata_obj: MetaData):
```
Comment (cbornet) [inline src/databricks/sqlalchemy/test_local/e2e/test_basic.py:191]: When table comments are supported this test can be replaced by the sqlalchemy TCK table comment tests.

**Comment by cbornet** [comment]
Hi @susodapop, will you have time to review this soon ?

**Comment by susodapop** [comment]
Yep! Just getting back up to speed following vacation time away.

**Comment by cbornet** [comment]
Awesome, thanks !

File: src/databricks/sqlalchemy/_ddl.py [hunk 2]
```diff
@@ -39,17 +39,38 @@ def visit_identity_column(self, identity, **kw):
...
                 "Databricks dialect ignores SQLAlchemy's autoincrement semantics. Use explicit Identity() instead."
             )
 
-        return super().get_column_specification(column, **kwargs)
+        colspec = super().get_column_specification(column, **kwargs)
+        if column.comment is not None:
+            literal = self.sql_compiler.render_literal_value(
+                column.comment, sqltypes.STRINGTYPE
+            )
+            colspec += " COMMENT " + literal
```
Comment (susodapop) [inline src/databricks/sqlalchemy/_ddl.py:71]: We need some unit tests for these. Will add in a separate commit.

File: src/databricks/sqlalchemy/_parse.py [hunk 3]
```diff
@@ -354,6 +354,7 @@ def parse_column_info_from_tgetcolumnsresponse(thrift_resp_row) -> ReflectedColu
         "type": final_col_type,
         "nullable": bool(thrift_resp_row.NULLABLE),
         "default": thrift_resp_row.COLUMN_DEF,
+        "comment": thrift_resp_row.REMARKS,
```
Comment (susodapop) [inline src/databricks/sqlalchemy/_parse.py:357]: We should add unit tests for this also.

**Review by susodapop — COMMENTED** [review]
Thanks for this contribution! Looks great. I'm going to push a few test additions. Plus we need an entry in the changelog.

File: src/databricks/sqlalchemy/_ddl.py (hunk 2 above)
Comment (susodapop) [reply src/databricks/sqlalchemy/_ddl.py:71]: Done here [e512603](https://github.com/databricks/databricks-sql-python/pull/306/commits/e512603c7d71eb945e09337587470b9d3e6307f7)

**Comment by susodapop** [comment]
I've changed the base branch for this PR to a `sqlalchemy-staging` branch so that I can test these changes alongside #308 and update the changelog in one fell swoop.

File: src/databricks/sqlalchemy/_parse.py (hunk 3 above)
Comment (susodapop) [inline src/databricks/sqlalchemy/_parse.py:357]: In order to pass the `ComponentReflectionTest` we need to update this line to

```python
"comment": thrift_resp_row.REMARKS or None
```

**Pull request merged**